app/controllers/friend_requests_controller.rb

```diff
@@ -4,13 +4,7 @@ class FriendRequestsController < ApplicationController
 
 
   def current_user_sent_friend_requests
```

Comment:
I think it's better to rename this method as send_friend_requests

Reply:
current_user_sent_friend_requests (GET request) here means that the request I sent to other users.